hibernate-core/src/test/java/org/hibernate/id/SequenceStyleGeneratorBehavesLikeSequeceHiloGeneratorWitZeroIncrementSizeTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
```

Comment:
why do you have the duplicated license header?

You might want to check your templates, I had found other classes with the similar thing in the past (and have cleaned them up)

Reply:
I fixed the template, it seemed the line `#parse("File Licence.java")` is not anymore necessary

Reply:
removed also the duplicate header